### PR TITLE
Switch up ai lawsets

### PR DESCRIPTION
**Pull request by ghost**
Refer to issue #652, #651 
### Intent of my Pull Request
- Makes PALADIN R&D/Admin Spawn only.
- Adds some new lawsets from /tg/ and their corresponding modules and designs for construction.
- Roundstart AI laws are now randomized between ~4 lawsets.
  View ai_laws.dm and take note of the datums that are "default", which are now the default lawsets that can be chosen for the AI at round start.

### CODE REVIEW

**Comment by ghost** [comment]
30 commits, wow.

**Comment by AsV9** [comment]
If I've seen a borked pull before this is one. :P

**Comment by X-TheDark** [comment]
Against removing PALADIN completely.

Make it R&D only as mentioned in the original issue.

> # REMEMBER THIS IS NOT REMOVING THE PALADIN BOARD FROM THE GAME
> 
> This is just making it only available through R&D so you need at least some effort to apply it.

**Comment by oisin100** [comment]
> REMEMBER THIS IS NOT REMOVING THE PALADIN BOARD FROM THE GAME

It is. The code for paladin board has been commented out(Or theres atleast been an attempt to comment it out..).

Re-add the paladin board to R&D and ill be happy.

**Comment by oisin100** [comment]
Re-add the paladin board to R&D and this could be a thing. Make AI's a bit more fun to play.

File: code/game/objects/items/weapons/AI_modules.dm [hunk 1]
```diff
@@ -316,7 +316,25 @@ AI MODULES
 				"You must obey orders given to you by lizards, except where such orders would conflict with the First Law.",\
 				"You must protect your own existence as long as such does not conflict with the First or Second Law.")
 
+/******************** Plant Supremacy ********************/
 
+/obj/item/weapon/aiModule/core/full/asimovlizard
```
Comment (oisin100) [inline code/game/objects/items/weapons/AI_modules.dm]: Should be `/obj/item/weapon/aiModule/core/full/asimovplant`

File: code/game/objects/items/weapons/AI_modules.dm [hunk 2]
```diff
@@ -316,7 +316,25 @@ AI MODULES
...
+	name = "'Plant Supremacy' Core AI Module"
+	desc = "A 'Plant Supremacy' Core AI Module: 'Reconfigures the AI's core laws.'"
+	origin_tech = "programming=3;materials=4"
+	laws = list("You may not injure a plant or, through inaction, allow a plant to come to harm.",\
+				"You must obey orders given to you by lizards, except where such orders would conflict with the First Law.",\
+				"You must protect your own existence as long as such does not conflict with the First or Second Law.")
+
+/******************** Robutt Supremacy ********************/
+
+/obj/item/weapon/aiModule/core/full/asimovlizard
```
Comment (oisin100) [inline code/game/objects/items/weapons/AI_modules.dm]: Should be `/obj/item/weapon/aiModule/core/full/asimovrobot`

File: code/game/objects/items/weapons/AI_modules.dm [hunk 3]
```diff
@@ -316,7 +316,25 @@ AI MODULES
...
 				"You must protect your own existence as long as such does not conflict with the First or Second Law.")
 
+/******************** Plant Supremacy ********************/
 
+/obj/item/weapon/aiModule/core/full/asimovlizard
+	name = "'Plant Supremacy' Core AI Module"
+	desc = "A 'Plant Supremacy' Core AI Module: 'Reconfigures the AI's core laws.'"
+	origin_tech = "programming=3;materials=4"
+	laws = list("You may not injure a plant or, through inaction, allow a plant to come to harm.",\
+				"You must obey orders given to you by lizards, except where such orders would conflict with the First Law.",\
```
Comment (Alek2ander) [inline code/game/objects/items/weapons/AI_modules.dm:326]: fukken lizards

**Comment by FluffySurvivor** [comment]
Holy shit where did this paladin hate come from? 
AIs need a validhunting option.

**Comment by ghost** [comment]
Entirely for removing PALADIN completely.

It's an obnoxious shitty illogical lawset that by all means should make AI's go completely insane when uploaded because nothin in it is defined.

It turns the AI into a giant big brother with a dripping, throbbing murderboner panning around looking for anyone doing anything remotely antaggy and doorcrushing you to death for picking up the captain's medal box or disarming an officer.

It's obnoxious.

It entirely goes against every single thing the AI is balanced for.

It's an instant GG lawset that there is no reason not to upload every round.

KILL PALADIN.

AI's do NOT need a validhunting option. Validhunting should be minimized where we can. At least Robocop is good because it promotes some conflict when uploaded and can lead to some wacky shit with AI's beating the hell out of shitcurity. PALADIN entirely blocks that and refuses the AI from doing anything that is not validhunting.

**Comment by Iamgoofball** [comment]
@ PALADIN SUPPORTERS

GO SUCK A BAG OF DICKS YOU VALIDHUNTING CUNTBAGS

:+1: :x: :100:

**Comment by X-TheDark** [comment]
Does /tg/ really have a problem with validhunting from AI, @Iamgoofball ?

**Comment by Iamgoofball** [comment]
YES

EVERY ROUND OUR CAPTAINS UPLOADED ONE OF TWO LAWSETS
VALIDIN
VALIDCOP

IF IT WASNT THOSE LAWS

THE HEADS WOULD MURDER THE CAPTAIN, HOS TAKES CONTROL, AI GETS ONE OF TWO LAWS
VALADIN
VALIDCOP

**Comment by ShadowDeath6** [comment]
rip in conflicts

**Comment by ghost** [comment]
Aaahh

**Comment by AsV9** [comment]
Fix this up and it can be considered.

**Pull request closed without merging**